Source/reactions/Castro_react.cpp

```diff
@@ -20,7 +20,7 @@ Castro::do_old_reactions (Real time, Real dt)
     advance_status status {};
 
 #ifndef SIMPLIFIED_SDC
-    bool burn_success = true;
+    int burn_success{1};
```

Comment:
We should also then change the `if (!burn_success)` checks

Reply:
fixed